app/javascript/stylesheets/components/_title.scss

```diff
@@ -0,0 +1,12 @@
+.welcome-title {
+  color: $dark-blue;
+  display: flex;
+  align-items: center;
+  margin-bottom: 20px;
+
+  h1 {
+  font-size: 44px;
+  line-height: 60px;
+  font-weight: 700;
+  }
```

Comment:
On n'est pas censé toucher à la taille des `h1...6` qui sont des sortes de presets, il vaudrait mieux utiliser un smple `p` ici.

Reply:
Je suis pas d'accord avec ça. Les `h1...6` ont aussi une valeur déclarative pour les moteurs de recherche par exemple, et le h1 est primordial pour le référencement. Mais même au delà du référencement, si tu fais un titre, tu es sensé utilisé une des balises h (que ce soit pour les lecteurs de page, pour la logique, etc). 
C'est aussi assez standard de faire des presets persos pour les `h1...6` d'un site de ce que j'en sais, mais en l'occurrence ce n'est même pas ce que je fais, pour éviter de toucher aux presets justement je change juste les `h1` de la classe `.welcome-title`

Reply:
Effectivement c'est discutable et les avis divergent sur la question (https://www.quora.com/Is-it-bad-practice-to-change-the-font-size-for-headers-h1-to-h6-tags-Should-I-just-use-the-p-tag-instead-and-change-its-font-size). Par contre ma proposition de remplacer par un `p` ne semble pas être bonne. 
Ok pour laisser comme ça pour l'instant.

Reply:
Mais ça me va de garder les presets pour les `h1...6` par défaut, j'avais d'ailleurs pris en compte ta remarque (que tu m'avais déjà formulé précédemment) en prenant soin de redéfinir le `h1` que pour la classe `welcome-title`